docs/skopeo.1.md

```diff
@@ -18,6 +18,10 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   default read from `$HOME/.kube/config`, which is set e.g. using
   `(oc login)`.
 
+  **container-storage://**_path_**:**_tag_
```

Comment:
The actual formats are, AFAICS:

- (an optional `[`root`]` or `[`driver`@`root`]` prefix. Supposedly that isn’t sufficient to make the choices work, so we can ignore it here.)
- one of:
   - dockerlikeReference
   - dockerlikeReference`@`algo`:`digest , where algo`:` is optional (defaulting to `sha256:`)
   - `@`algo`:`digest , where algo`:` is optional (defaulting to `sha256:`)

dockerlikeReference is not a path, it uses the `docker.io/library` normalization. (and “digest” is a containers/storage ID, not a manifest digest)